### DISCUSSION TITLE
Where to find the scan result

### DISCUSSION THREAD

**Discussion by Zinou16** [linkchecker/linkchecker#723] [Q&A]
Hello,

I want to run linkchecker and output the result to a file (Ex: html format)
what's are the parameter to include and where to find the output file?

Regards,

**Comment by cjmayo** [comment]
`-F` is the option, for more info including the output filename see:

https://linkchecker.github.io/linkchecker/man/linkchecker.html#cmdoption-F

**Comment by Zinou16** [comment]
Thanks for the reply, but the doc is unclear and does not give examples!
Does this mean put -F followed by the filename + extension ?

**Comment by cjmayo** [comment]
The man page convention is that square brackets denote optional fields. So, if you want to specify a filename but no encoding it is saying:

`-F TYPE/FILENAME`

e.g. `linkchecker -F html/my.html http://localhost`
